build_tools/cmake/iree_c_module.cmake

```diff
@@ -101,7 +101,7 @@ function(iree_c_module)
       "${_TESTONLY_ARG}"
     DEPS
       # Include paths and options for the runtime sources.
-      iree::runtime::src::defs
+      iree::defs
```

Comment:
Why elide runtime for this target?

Reply:
It's actually the other way: this was the only one that had "runtime" in it because it didn't line up with the directory tree (accidentally). We may want to add a runtime namespace to this tree at some point, but we'd need to change them all, not just have this one be weird.